diff --git a/Jellyfin.Data/UserEntityExtensions.cs b/Jellyfin.Data/UserEntityExtensions.cs
index 0fc8d3cd25..dfcc4f27a1 100644
--- a/Jellyfin.Data/UserEntityExtensions.cs
+++ b/Jellyfin.Data/UserEntityExtensions.cs
@@ -185,6 +185,7 @@ public static class UserEntityExtensions
         entity.Permissions.Add(new Permission(PermissionKind.EnableSyncTranscoding, true));
         entity.Permissions.Add(new Permission(PermissionKind.EnableAudioPlaybackTranscoding, true));
         entity.Permissions.Add(new Permission(PermissionKind.EnableLiveTvAccess, true));
+        entity.Permissions.Add(new Permission(PermissionKind.EnableLiveTvMultiview, true));
         entity.Permissions.Add(new Permission(PermissionKind.EnableLiveTvManagement, false));
         entity.Permissions.Add(new Permission(PermissionKind.EnableSharedDeviceControl, true));
         entity.Permissions.Add(new Permission(PermissionKind.EnableVideoPlaybackTranscoding, true));
diff --git a/Jellyfin.Server.Implementations/Users/UserManager.cs b/Jellyfin.Server.Implementations/Users/UserManager.cs
index 8198ebce47..59a1857017 100644
--- a/Jellyfin.Server.Implementations/Users/UserManager.cs
+++ b/Jellyfin.Server.Implementations/Users/UserManager.cs
@@ -456,6 +456,7 @@ namespace Jellyfin.Server.Implementations.Users
                     EnableRemoteAccess = user.HasPermission(PermissionKind.EnableRemoteAccess),
                     EnableLiveTvManagement = user.HasPermission(PermissionKind.EnableLiveTvManagement),
                     EnableLiveTvAccess = user.HasPermission(PermissionKind.EnableLiveTvAccess),
+                    EnableLiveTvMultiview = user.HasPermission(PermissionKind.EnableLiveTvMultiview),
                     EnableMediaPlayback = user.HasPermission(PermissionKind.EnableMediaPlayback),
                     EnableAudioPlaybackTranscoding = user.HasPermission(PermissionKind.EnableAudioPlaybackTranscoding),
                     EnableVideoPlaybackTranscoding = user.HasPermission(PermissionKind.EnableVideoPlaybackTranscoding),
@@ -843,6 +844,7 @@ namespace Jellyfin.Server.Implementations.Users
                     user.SetPermission(PermissionKind.EnableRemoteAccess, policy.EnableRemoteAccess);
                     user.SetPermission(PermissionKind.EnableLiveTvManagement, policy.EnableLiveTvManagement);
                     user.SetPermission(PermissionKind.EnableLiveTvAccess, policy.EnableLiveTvAccess);
+                    user.SetPermission(PermissionKind.EnableLiveTvMultiview, policy.EnableLiveTvMultiview);
                     user.SetPermission(PermissionKind.EnableMediaPlayback, policy.EnableMediaPlayback);
                     user.SetPermission(PermissionKind.EnableAudioPlaybackTranscoding, policy.EnableAudioPlaybackTranscoding);
                     user.SetPermission(PermissionKind.EnableVideoPlaybackTranscoding, policy.EnableVideoPlaybackTranscoding);
diff --git a/Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs b/Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs
index a498901481..0f19785310 100644
--- a/Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs
+++ b/Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs
@@ -75,6 +75,7 @@ namespace Jellyfin.Server.Extensions
                 options.AddPolicy(Policies.FirstTimeSetupOrIgnoreParentalControl, new FirstTimeSetupRequirement(false, false));
                 options.AddPolicy(Policies.IgnoreParentalControl, new DefaultAuthorizationRequirement(validateParentalSchedule: false));
                 options.AddPolicy(Policies.LiveTvAccess, new UserPermissionRequirement(PermissionKind.EnableLiveTvAccess));
+                options.AddPolicy(Policies.LiveTvMultiview, new UserPermissionRequirement(PermissionKind.EnableLiveTvMultiview));
                 options.AddPolicy(Policies.LiveTvManagement, new UserPermissionRequirement(PermissionKind.EnableLiveTvManagement));
                 options.AddPolicy(Policies.LocalAccessOrRequiresElevation, new LocalAccessOrRequiresElevationRequirement());
                 options.AddPolicy(Policies.SyncPlayHasAccess, new SyncPlayAccessRequirement(SyncPlayAccessRequirementType.HasAccess));
diff --git a/MediaBrowser.Common/Api/Policies.cs b/MediaBrowser.Common/Api/Policies.cs
index 435f4798ff..298c57098b 100644
--- a/MediaBrowser.Common/Api/Policies.cs
+++ b/MediaBrowser.Common/Api/Policies.cs
@@ -80,6 +80,11 @@ public static class Policies
     /// </summary>
     public const string LiveTvAccess = "LiveTvAccess";
 
+    /// <summary>
+    /// Policy name for using LiveTV Multiview.
+    /// </summary>
+    public const string LiveTvMultiview = "LiveTvMultiview";
+
     /// <summary>
     /// Policy name for managing LiveTV.
     /// </summary>
diff --git a/MediaBrowser.Model/Users/UserPolicy.cs b/MediaBrowser.Model/Users/UserPolicy.cs
index 95e4d46c59..76b550228f 100644
--- a/MediaBrowser.Model/Users/UserPolicy.cs
+++ b/MediaBrowser.Model/Users/UserPolicy.cs
@@ -32,6 +32,7 @@ namespace MediaBrowser.Model.Users
             ForceRemoteSourceTranscoding = false;
             EnableLiveTvManagement = true;
             EnableLiveTvAccess = true;
+            EnableLiveTvMultiview = true;
 
             // Without this on by default, admins won't be able to do this
             // Improve in the future
@@ -133,6 +134,8 @@ namespace MediaBrowser.Model.Users
 
         public bool EnableLiveTvAccess { get; set; }
 
+        public bool EnableLiveTvMultiview { get; set; }
+
         public bool EnableMediaPlayback { get; set; }
 
         public bool EnableAudioPlaybackTranscoding { get; set; }
diff --git a/src/Jellyfin.Database/Jellyfin.Database.Implementations/Enums/PermissionKind.cs b/src/Jellyfin.Database/Jellyfin.Database.Implementations/Enums/PermissionKind.cs
index 0818639636..360c96350e 100644
--- a/src/Jellyfin.Database/Jellyfin.Database.Implementations/Enums/PermissionKind.cs
+++ b/src/Jellyfin.Database/Jellyfin.Database.Implementations/Enums/PermissionKind.cs
@@ -124,4 +124,9 @@ public enum PermissionKind
     /// Whether the user can edit lyrics.
     /// </summary>
     EnableLyricManagement = 23,
+
+    /// <summary>
+    /// Whether the user can use Live TV Multiview.
+    /// </summary>
+    EnableLiveTvMultiview = 24,
 }
